docs/_docs/extensions-and-integrations/change-data-capture-extensions.adoc

```diff
@@ -175,6 +183,16 @@ Kafka to Ignite configuration file should contain the following beans that will
 | `threadCount` | Count of threads to proceed consumers. Each thread poll records from dedicated partitions in round-robin manner. | 16
 |===
 
+`kafkaRequestTimeout` property sets timeouts for a following `KafkaConsumer` operations:
+
+- Poll from event and metadata topics.
```

Comment:
Do we actually need list of these operations? Is there any kafka request that doesn't use this timeout?

Reply:
Timeouts for `#poll` are, of cause, is on the surface. But I think, that we can at least tell about request types, that they are affected by a timeout, because they can (unlike `#poll`) raise timeout exceptions.